src/cljdoc/render/api.clj

```diff
@@ -32,12 +32,14 @@
       [nil m])))
 
 (defn docstring->html [doc-str render-wiki-link]
-  [:div.lh-copy.markdown
-   (-> doc-str
-       (rich-text/markdown-to-html
+  [:div
+   [:div.lh-copy.markdown
+    (-> doc-str
+        (rich-text/markdown-to-html
          {:escape-html? true
           :render-wiki-link (comp render-wiki-link parse-wiki-link)})
-       hiccup/raw)])
+        hiccup/raw)]
+   [:pre.lh-copy.raw.hide (hiccup/raw doc-str)]])
```

Comment:
`hiccup/raw` will prevent escaping of HTML in the passed string, I think in this case we do want any HTML in the doctoring to be escaped.

Reply:
I removed the call which should escape html